app/component/CustomizeSearch.js

```diff
@@ -304,6 +396,41 @@ class CustomizeSearch extends React.Component {
         router: this.context.router,
       },
     );
+    this.setState({
+      [name]: value && mapToSlider(value, sliderValues),
+    });
+  }
+
+  resetParameters = () => {
+    resetCustomizedSettings();
+    this.setState({
+      walkSpeed: defaultSettings.walkSpeed &&
+      mapToSlider(defaultSettings.walkSpeed, this.walkingSpeedSliderValues),
```

Comment:
A minor issue: defaultSettings is our const data, so value check at start is not necessary. We can remove

  `defaultSettings.walkSpeed &&`

etc.